compiler/accelerators/snax.py

```diff
@@ -93,6 +93,7 @@ class SNAXPollingBarrier(Accelerator, ABC):
 
     @staticmethod
     def lower_acc_await(acc_op: accfg.AcceleratorOp) -> Sequence[Operation]:
+        nops = [llvm.InlineAsmOp("nop", "", [], [], has_side_effects=True) for _ in range(20)]
```

Comment:
I found some tests to change their amount of errors by just changing this range alone.
10000 nops does the trick. At least sometimes. @rgantonio is the HPWE simulation deterministic?